elements/pfe-text/demo/index.html

```diff
@@ -33,8 +33,8 @@ <h1>&lt;pfe-text&gt;</h1>
 
        <pfe-band pfe-color="lighter">
         <pfe-text slot="pfe-band--header">
-          <p pfe-text="headline-2">Band title: lorem ipsum</strong></p>
```

Comment:
@starryeyez024 I wanted to get your take on this idea.  What if we abstract the attributes away from numbers to avoid confusion with markup and keep with the alpha, beta, gamma, naming?

Reply:
I can jive with that :)